src/sentry/static/sentry/app/views/issueList/createSavedSearchButton.tsx

```diff
@@ -80,12 +89,12 @@ class CreateSavedSearchButton extends React.Component {
     }
   };
 
-  handleChangeName = val => {
-    this.setState({name: val});
+  handleChangeName = (val: string | number | boolean) => {
+    this.setState({name: val as string});
   };
 
-  handleChangeQuery = val => {
-    this.setState({query: val});
+  handleChangeQuery = (val: string | number | boolean) => {
+    this.setState({query: val as string});
```

Comment:
```suggestion
    this.setState({query: String(val)});
```
Will not only appease typescript but will improve runtime type safety as well.